packages/manager/apps/dedicated/client/app/otrs/otrs.controller.js

```diff
@@ -1,293 +1,38 @@
-import debounce from 'lodash/debounce';
-import forEach from 'lodash/forEach';
-import find from 'lodash/find';
-import includes from 'lodash/includes';
-import isDate from 'lodash/isDate';
-import map from 'lodash/map';
-import set from 'lodash/set';
+import { TICKET_STATE, TICKET_LAST_MESSAGE_FROM } from './otrs.constants';
 
-export default /* @ngInject */ function otrsCtrl(
-  $rootScope,
-  $scope,
-  $location,
-  $translate,
-  OtrsPopupService,
-  Otrs,
-  Alerter,
-  coreConfig,
-  OTRS_POPUP_UNIVERSES,
-) {
-  let firstLoading = false;
-
-  $scope.itemsPerPage = 10;
-  $scope.today = new Date();
-  $scope.minCreationDateSearch = {
-    opened: false,
-  };
-  $scope.maxCreationDateSearch = {
-    opened: false,
-  };
-  $scope.worldPart = coreConfig.getRegion();
-  $scope.showFilters = false;
-  $scope.currentPage =
-    $location.search() && $location.search().currentPage != null
-      ? parseInt($location.search().currentPage, 10)
-      : 1;
-  $scope.selectedLanguage = $scope.selectedLanguage || { value: 'fr_FR' };
-
-  function loadSearch() {
-    if (!sessionStorage) {
-      return;
-    }
-    if (sessionStorage.getItem('OVH_SUPPORT_FILTERS')) {
-      $scope.search = JSON.parse(sessionStorage.getItem('OVH_SUPPORT_FILTERS'));
-      $scope.showFilters = true;
-    }
+export default class otrsCtrl {
```

Comment:
class start with a Capital letter:
```suggestion
export default class OtrsCtrl {
```